src/courseware/course/CourseBreadcrumbs.jsx

```diff
@@ -60,7 +61,7 @@ export default function CourseBreadcrumbs({
   }, [courseStatus, sequenceStatus]);
 
   return (
-    <nav aria-label="breadcrumb" className="my-4">
+    <nav aria-label="breadcrumb" className={classNames('my-4 d-inline-block col-sm-10')}>
```

Comment:
```suggestion
    <nav aria-label="breadcrumb" className="my-4 d-inline-block col-sm-10">
```
If you're putting one string into classNames, may as well not use it.

Reply:
Removed! I originally had a conditional and forgot to remove it